src/main/scala/me/rexim/issuestant/Permalink.scala

```diff
@@ -0,0 +1,26 @@
+package me.rexim.issuestant
+
+import scalaz.concurrent.Task
+import me.rexim.issuestant.github.EventsSource
+import me.rexim.issuestant.github.model._
+
+// TODO(1cdf1e3c-d19b-493d-9d33-f88c92ee24e3): Implement Permalink
+
+/** Issuestant service for Permalink protocol (see doc/Permalink.md)
+  *
+  * The class is stateless. The next state is returned by update()
+  * method. The state should be managed by the invoker.
+  *
+  * @param eventSource the source of GitHub events
+  */
+class Permalink(eventSource: EventsSource) {
```

Comment:
Disable coverage for this class. See https://github.com/scoverage/sbt-scoverage#exclude-classes-and-packages